### ISSUE TITLE
coveralls 0.6.0+2 hanging on run

### ISSUE THREAD

**Issue by pq** [block-forest/dart-coveralls#84]
Builds using `coveralls` are green again but unfortunately are timing out.

https://travis-ci.org/dart-lang/linter/jobs/407163523

```
Activated dart_coveralls 0.6.0+2.
Bad state: Failed to complete within 60s
```

(Confirmed locally.)

**Comment by pq** [comment]
Very likely new async semantics related.

**Comment by pq** [comment]
Kudos to @denniskaselow.  Thanks to the fix in #80 (and published in `0.6.0+3`), we've got our first coverage runs on the linter in a month.

![image](https://user-images.githubusercontent.com/67586/43167736-9243fdfe-8f4f-11e8-9815-1bfa4d03a6ec.png)

🎉
